### PR TITLE
[13.0][MIG] website_no_crawler: Migration to 13.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
cc @Tecnativa TT25962

### CODE REVIEW

**Review by joao-p-marques — APPROVED** [review]
LGTM :+1:

File: website_no_crawler/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2016 B-informed B.V.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Alter robots.txt disallow indexing",
+    "summary": "Disables robots.txt for indexing by webcrawlers like Google",
+    "license": "AGPL-3",
+    "version": "13.0.1.0.0",
+    "website": "https://github.com/OCA/website",
+    "author": "B-Informed B.V., Odoo Community Association (OCA)",
```
Comment (victoralmau) [inline website_no_crawler/__manifest__.py:9]:
```suggestion
    "author": "B-Informed B.V., Tecnativa, Odoo Community Association (OCA)",
```

**Review by victoralmau — CHANGES_REQUESTED** [review]
Everything ok but add Tecnativa in manifest file, please.

**Comment by yajo** [comment]
@pedrobaeza are we really gonna need this? These days [doodba configures traefik easily to disallow crawlers automatically](https://github.com/Tecnativa/doodba-copier-template/blob/4b85e88858b2e56aa1be25adacdd5d9953897198/copier.yml#L271-L284)...

File: website_no_crawler/__manifest__.py (hunk 1 above)
Comment (pedrobaeza) [reply website_no_crawler/__manifest__.py:9]: Not really if the contribution is very little.

**Comment by pedrobaeza** [comment]
Is such configuration retained in copier template. If so, let's switch to that one. Anyway, this module can let migrated although we don't use it anymore, as the work has been done.

**Review by yajo — APPROVED** [review]
> Is such configuration retained in copier template

Yes.



> the work has been done

True haha.

**Comment by pedrobaeza** [comment]
Please check instances that have this module and make the switch.

**Pull request merged**